sql/create.sql

```diff
@@ -166,8 +166,6 @@ CREATE TABLE IF NOT EXISTS `PREFIX_ec_ecopresto_tracking` (
   `url_exp` varchar(255) NOT NULL,
   PRIMARY KEY (`id`)
 ) ENGINE=MyISAM DEFAULT CHARSET=utf8;
-
-

```

Comment:
PREFIX_ec_ecopresto_cataloguebrut is not need anymore (suppress function  setCatalogCSVMySQL_Parsephp which is not used anymore)